### PR TITLE
feat: Do not show notification for opened conversation

### PR DESCRIPTION
**Pull request by borichellow**
# What's new in this PR?

## [Task 1854](https://wearezeta.atlassian.net/browse/AR-1854)

- Do not show notifications for the conversation that is currently opened.
- added `CurrentScreenManager`, it observes the currently opened screen, or if the app was hidden.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-753/1/) **failed**.

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-753/2/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3856720)(3856720)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3856720)(3856720)-dev-release.apk)

**Review by alexandreferris — APPROVED** [review]
great work @borichellow !

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-753/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(3866360)(3866360)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(3866360)(3866360)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/util/CurrentScreenManager.kt [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                NavigationItem.Conversation -> {
+                    arguments?.getString(EXTRA_CONVERSATION_ID)
+                        ?.parseIntoQualifiedID()
+                        ?.let { Conversation(it) }
+                        ?: SomeOther
+                }
+                else -> SomeOther
+            }
+    }
+}
```
Comment (gongracr) [inline app/src/main/kotlin/com/wire/android/util/CurrentScreenManager.kt:74]: Can we extract to separated file?

**Review by gongracr — APPROVED** [review]
Beautifully executed! 🦁 🍨 just a small comment about code separation

**Pull request merged**